src/dashboards/Compact/styles.scss

```diff
@@ -14,6 +14,14 @@
             overflow-x: hidden;
             margin-bottom: 1rem;
         }
+
+        .scootertile {
+            width: 100%;
+            padding-top: 2rem;
+            overflow-y: hidden;
+            overflow-x: hidden;
```

Comment:
Du kan slå desse sammen til `overflow: hidden`

Reply:
De to overflow-ene eller alle fire? 🤯

Reply:
overflowene :)